rust/operator-binary/src/security/kerberos.rs

```diff
@@ -52,6 +52,11 @@ impl HdfsSiteConfigBuilder {
     fn add_wire_encryption_settings(&mut self) -> &mut Self {
         self.add("dfs.data.transfer.protection", "privacy");
         self.add("dfs.encrypt.data.transfer", "true");
+        self.add(
+            "dfs.encrypt.data.transfer.cipher.suite",
+            "AES/CTR/NoPadding",
+        );
+        self.add("dfs.encrypt.data.transfer.cipher.key.bitlength", "128");
```

Comment:
I don't think that's needed as it's the default anyway and I think I'd prefer keeping the default values so it moves with the times should Hadoop update its default.

@Jimvin - any opinions?